ansible/roles/operator-pipeline/templates/openshift/tasks/acquire-lock.yaml

```diff
@@ -83,3 +83,4 @@ spec:
         create_lease_yaml
         # Acquire Lease
         acquire_lease
+      timeout: "0h35m0s"
```

Comment:
This timeout might be in conflict with a timeout given by the task parameter. Is this needed if the task itself waits for a lease with a timeout already?